### PR TITLE
clarify roles and permissions based on feedback

### PR DESCRIPTION
**Pull request by ultrasaurus**
this addresses feedback  on https://github.com/cncf/sig-security/pull/219 by @TheFoxAtWork

### CODE REVIEW

File: assessments/guide/security-reviewer.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-provide an independent analysis and estimation of the above information. 
-The goal is to ask questions to understand if there are hidden assumptions,
-underestimated risk, or design issues that harm the security of the project. 
+Reviewers seek to understand the system and probe its security.
+Design level issues should be considered as well as any high
+level problems with the project’s setup and operation. The [process](./)
+is intended to provide an independent analysis based primarily on documents
+provided by the project itself.
+
+Reviewers may look at code, but aren't required to do so. They review docimen
```
Comment (TheFoxAtWork) [inline assessments/guide/security-reviewer.md]: typo?

File: governance/roles.md [hunk 2]
```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
...
-described below.
+Members often contribute by working in small groups that research and discuss
+options and then share their findings with the rest of the group in a
+presentation and typically contribute to the body of work in this
+repository (via Pull Request). Some tasks can be effectively executed
+by simply chiming in on a github issue and independently contributing a PR
+without any particular role or authorization.
+
+This section describes roles where more coordination is helpful and the
+work benefits from identifying one or more Members who will take
```
Comment (TheFoxAtWork) [inline governance/roles.md]: Notice the capitalization of "members" on line 107 and 108, is this intentional?  if so we need to ensure we are consistent in our other docs too.

File: governance/roles.md [hunk 3]
```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
...
+on specific responsibilities.  Our goal is to empower Members to move forward
+independently, while defining an approval process designed to support
+communication and alignment across the wider group.
+
+New roles in this section are typically defined after someone has acted in
+that role for some time informally, then proposed as an on-going role by Pull
+Request which must be approved by a majority of Chairs.  Process clarifications
+are welcome and PRs can be approved by any single Chair.
```
Comment (TheFoxAtWork) [inline governance/roles.md]: same comment on "chairs" for line 114 and 115 as "members"  we'll need to update contributions guide if this is how these roles should be written.

File: governance/roles.md [hunk 4]
```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
...
+
+Members who take on Roles are responsible for
+understanding the approval policies related to their role and ensure that
+commits to the repo are reviewed in accordance with our [governance](./)
```
Comment (TheFoxAtWork) [inline governance/roles.md]: not sure if "governance" is the same as the approval policies in line 118, if so we should be more clear "...the approval policies, as defined in our [governance](./),

File: governance/roles.md [hunk 5]
```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
...
+
+**Write access to the repo**: Where applicable, the following roles define
+specific areas of the repo or actions on issues where changes require write
+access. In any case, governance is not enforced by access permissions, but
+rather by Members who are expected to thoughtfully consider their actions to
+support the group.
+
+**Github permissions note**: Facilitation roles are identified in
+[github settings](/.github/settings.yaml) which grants Github permissions.
+Pull Requests to appoint Members to new Roles must be approved by at least one
```
Comment (TheFoxAtWork) [inline governance/roles.md]: "Roles" on line 130 capitalized in line.  seems weird even with the "Members" and "Chair" changes

File: governance/roles.md (hunk 2 above)
Comment (ultrasaurus) [reply governance/roles.md]: made it consistent with rest of doc that "member" is lower case.  (Note that "role" capitalization is inconsistent, but don't want to expand the scope of this PR)

File: governance/roles.md (hunk 4 above)
Comment (ultrasaurus) [reply governance/roles.md]: well, the security reviewer role is defined outside of /governance -- just trying to say that anyone with a role that involves commits to the repo is responsible for anything that is written in our governance (e.g. consistent with charter and mission, transparency, inclusion, and anything else that we wrote down as overall policy for how we work together as a group)

Feel free to suggest alternate wording, but I don't want to imply less responsibility for people who take on roles.  I want people to feel an obligation to fully understand and commit to taking ownership of understanding our overall governance and acting appropriately.

File: governance/roles.md (hunk 5 above)
Comment (ultrasaurus) [reply governance/roles.md]: I will open a separate issue for consistent capitalization of Roles and Chair

File: governance/roles.md (hunk 2 above)
Comment (ultrasaurus) [reply governance/roles.md]: wrote up separate issue for capitalization consistency of Roles and Chair https://github.com/cncf/sig-security/issues/231

File: governance/roles.md (hunk 4 above)
Comment (ultrasaurus) [reply governance/roles.md]: maybe I'll pull this out as an issue, so we can get the rest of the PR improvements committed...  if other folks agree this PR is net-positive

Comment (ultrasaurus) [reply governance/roles.md]: see https://github.com/cncf/sig-security/issues/233

**Pull request merged**